scheduler/config.edn

```diff
@@ -40,7 +40,11 @@
        :keystore-path #config/env "COOK_KEYSTORE_PATH"
        :keystore-type "pkcs12"
        :keystore-pass "cookstore"}
- :rate-limit {:user-limit-per-m 1000000}
+ :rate-limit {:rate-limit-expire-min 120 ; Expire unused rate limit entries after 2 hours.
```

Comment:
`rate-limit-expire-min` can become `expire-min` now that it is inside the `rate-limit` entry